cocotb/__init__.py

```diff
@@ -128,6 +128,14 @@ def _reopen_stream_with_buffering(stream_name):
 _library_coverage = None
 """ used for cocotb library coverage """
 
+top = None  # type: SimHandleBase
+"""
+A handle to the :envvar:`TOPLEVEL` entity. ``None`` if :mod:`cocotb` was not loaded from a simulator.
+This is equivalent to the :term:`DUT` parameter given to cocotb tests, so it can be used wherever that variable can be used.
+It is particularly useful for extracting information about the :term:`DUT` in module-level class and function definitions;
+and in parameters to :class:`~cocotb.regression.TestFactory`s.
```

Comment:
```suggestion
A handle to the :envvar:`TOPLEVEL` object.

This is equivalent to the :term:`DUT` parameter given to cocotb tests, so it can be used wherever that variable can be used.
It is particularly useful for extracting information about the :term:`DUT` in module-level class and function definitions;
and in parameters to :class:`~cocotb.regression.TestFactory`s.
``None`` if :mod:`cocotb` was not loaded from a simulator.
```
Separating the first line with a blank line will make this look better in e.g. IDE tooltips.
I'm not sure if the "object" in the first line is good, but "entity" might lead someone into thinking this is for VHDL only.

Reply:
Could use either "entity/module" or "entity or module"

Reply:
Is it possible for `TOPLEVEL` to be something other than an entity or module?

Reply:
A ``configuration`` in VHDL or ``config`` in Verilog (the latter being rather rare).